torch_frame/data/stats.py

```diff
@@ -81,9 +81,9 @@ def compute_col_stats(
 
     if stype == torch_frame.numerical:
         ser = ser.mask(ser.isin([np.inf, -np.inf]), np.nan)
-    ser = ser.dropna()
 
     return {
-        stat_type: stat_type.compute(ser, sep)
+        stat_type: (np.nan if ser.isnull().all() else stat_type.compute(
+            ser.dropna(), sep))
```

Comment:
Add test for the new case.

Reply:
got it, thanks @weihua916. 
![image](https://github.com/pyg-team/pytorch-frame/assets/70596906/ec928801-8e10-48ff-89b7-abe25c277df1)

Reply:
Thanks!